mne/preprocessing/ica.py

```diff
@@ -1264,38 +1286,75 @@ def find_bads_ref(self, inst, ch_name=None, threshold=3.0, start=None,
 
         Notes
         -----
-        Detection is based on Pearson correlation between the MEG data
-        components and MEG reference components.
-        Thresholding is based on adaptive z-scoring. The above threshold
-        components will be masked and the z-score will be recomputed
-        until no supra-threshold component remains.
-
-        Recommended procedure is to perform ICA separately on reference
-        channels, extract them using .get_sources(), and then append them to
-        the inst using .add_channels(), preferably with the prefix REF_ICA so
-        that they can be automatically detected.
+        ICA decomposition on MEG reference channels is used to assess external
+        magnetic noise and remove it from the MEG. Two methods are supported:
+
+        With the "together" method, only one ICA fit is used, which
+        encompasses both MEG and reference channels together. Components which
+        have particularly strong weights on the reference channels may be
+        thresholded and marked for removal.
+
+        With "separate," selected components from a separate ICA decomposition
+        on the reference channels are used as a ground truth for identifying
+        bad components in an ICA fit done on MEG channels only. The logic here
+        is similar to an EOG/ECG, with reference components replacing the
+        EOG/ECG channels. Recommended procedure is to perform ICA separately
+        on reference channels, extract them using .get_sources(), and then
+        append them to the inst using .add_channels(), preferably with the
+        prefix REF_ICA so that they can be automatically detected.
+
+        Thresholding in both cases is based on adaptive z-scoring:
+        The above threshold components will be masked and the z-score will be
+        recomputed until no supra-threshold component remains.
+
+        Validation and further documentation for this technique can be found
+        in :footcite:`HannaEtAl2020`
```

Comment:
You will need to add the `References` section with a `.. footbibliography::`, see how we do it in other methods/functions